semantic_release_template.yaml

```diff
@@ -1,12 +1,12 @@
 name: screwdriver-cd/semantic-release
 version: '1.0'
-description: Template for semantic release
+description: Template for semantic release with Node 10 or higher
 maintainer: daolam112@gmail.com
```

Comment:
probably want to update this email

Reply:
we could use `screwdriver-oss@yahooinc.com`